test/e2e/e2e_suite_test.go

```diff
@@ -605,8 +605,8 @@ defaultHugepagesSize: "1G"`,
 			Expect(hwProvCondition).ToNot(BeNil())
 			testutils.VerifyStatusCondition(*hwProvCondition, metav1.Condition{
 				Type:    string(provisioningv1alpha1.PRconditionTypes.HardwareProvisioned),
-				Status:  metav1.ConditionUnknown,
-				Reason:  string(metav1.ConditionUnknown),
+				Status:  metav1.ConditionFalse,
+				Reason:  string(provisioningv1alpha1.CRconditionReasons.Unknown),
```

Comment:
We set the `Message` to `"Hardware provisioning is in progress"` - I wonder if we should initialize the reason to `InProgress`. Either way, we should probably maintain parity between the condition's message and reason.

Reply:
I agree. We've started processing the request, we've sent off the NAR, etc, it's "in progress". We should be setting the condition status accordingly because we've started the processing - not waiting until something else has finished its own processing and we see some response